### ISSUE TITLE
Initialisation of the AR and random walk

### ISSUE THREAD

**Issue by seabbs** [CDCgov/Rt-without-renewal#170]
We are currently initialising the AR model using independent initialisation parameters but for the random walk we use both a initial intercept and then add in additional variance. 

https://github.com/CDCgov/Rt-without-renewal/blob/f71e0f89c7c016f086f180aed0cccaaea682581e/EpiAware/src/EpiLatentModels/randomwalk.jl#L93-L94

Do we want to update either of these or make this more flexible?

**Comment by SamuelBrand1** [comment]
I think we want to keep the initial value and the latent process somewhat separate. For example, in this approach we can condition on \epsilon_t being = 0 but still make an inference on `rw_init` which easily recovers time invariant $R_0$ inference.

**Comment by seabbs** [comment]
I agree but that might be hard to communicate as it depends on fixing after the problem has been constructed?

**Comment by seabbs** [comment]
This would be improved by #372 as the init terms could be set to 0 using a `FixedIntercept` latent model (i.e without needing to use `fix`. At the moment both of these approaches assume that they need to initialise and we could introduce a mode that makes this optional or as suggested here decompose these into two models. I think I prefer the idea of making it optional (i.e a within-model feature as this seems less error prone but open to suggests)
